content/metadata.yaml

```
@@ -446,13 +446,13 @@ authors:
   - Center for Data-Driven Discovery, Children's Hospital of Philadelphia
   - Division of Neurosurgery, Children's Hospital of Philadelphia
 - github: awaanders
-  name: Angela Waanders
-  initials: AW
+  name: Angela J. Waanders
+  initials: AJW
   orcid: 0000-0002-0571-2889
   email: awaanders@luriechildrens.org
   contributions: Supervison
   affiliations:
-  - Department of Oncology, Ann & Robert H. Lurie Children’s Hospital of Chicago
+  - Division of Hematology, Oncology, NeuroOncology, and Stem Cell Transplant, Ann & Robert H Lurie Children’s Hospital of Chicago
```

Should this be:
```suggestion
  - Division of Hematology, Oncology, Neuro-Oncology, and Stem Cell Transplant, Ann & Robert H Lurie Children’s Hospital of Chicago
```

Based on: https://www.luriechildrens.org/en/doctors/waanders-angela/

---

oh yes, just copied and pasted from email. :) We can go with hyphen!